packages/docs/package.json

```
@@ -9,13 +9,16 @@
     "dist"
   ],
   "scripts": {
-    "build": "yarn vendor && yarn micromodal && yarn choices && yarn prism && hexo g",
+    "prebuild": "yarn changelog && yarn vendor && yarn micromodal && yarn choices && yarn prism",
+    "build": "hexo g",
+    "prestart": "yarn build && hexo s",
+    "start": "yarn build && hexo s",
```

These are now duplicate commands.

```suggestion
    "start": "yarn build && hexo s",
```